### PR TITLE
[MBL-16333][Student] -  Other Student E2E test extensions and stabilizations

### PR DESCRIPTION
**Pull request by kdeakinstructure**
Implement many test extensions and stabilizations.

refs: MBL-16333
affects: Student
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

File: apps/student/src/androidTest/java/com/instructure/student/ui/e2e/k5/ImportantDatesE2ETest.kt [hunk 1]
```diff
@@ -94,7 +94,7 @@ class ImportantDatesE2ETest : StudentTest() {
         importantDatesPage.assertItemNotDisplayed(testNotImportantAssignment.name)
 
         Log.d(STEP_TAG, "Assert that the count of the items (5) and the day strings are correct on the Important Dates page.")
```
Comment (kristofnemere) [inline apps/student/src/androidTest/java/com/instructure/student/ui/e2e/k5/ImportantDatesE2ETest.kt:96]: The count in the log is still 5

File: apps/student/src/androidTest/java/com/instructure/student/ui/e2e/k5/ImportantDatesE2ETest.kt [hunk 2]
```diff
@@ -115,7 +115,7 @@ class ImportantDatesE2ETest : StudentTest() {
         importantDatesPage.assertItemNotDisplayed(testNotImportantAssignment.name)
 
         Log.d(STEP_TAG, "Assert that the count of the items (5) and the day strings are correct on the Important Dates page after the refresh.")
```
Comment (kristofnemere) [inline apps/student/src/androidTest/java/com/instructure/student/ui/e2e/k5/ImportantDatesE2ETest.kt:117]: The count in the log is still 5

**Review by kristofnemere — APPROVED** [review]

File: apps/student/src/androidTest/java/com/instructure/student/ui/e2e/FilesE2ETest.kt [hunk 3]
```diff
@@ -190,6 +211,20 @@ class FilesE2ETest: StudentTest() {
 
         Log.d(STEP_TAG,"Assert that empty view is displayed after deletion.")
         fileListPage.assertViewEmpty()
+
+        Log.d(STEP_TAG, "Navigate back to global File List Page. Assert that the 'unfiled' folder has 0 items because we deleted the only item in it recently.")
+        Espresso.pressBack()
+        refresh() //TODO after this bugfix: https://instructure.atlassian.net/browse/MBL-16937?atlOrigin=eyJpIjoiNWJjODY1MTI4NDE0NGQxM2E3ZjBiYTQzZDdlM2IwOWIiLCJwIjoiaiJ9
+        fileListPage.assertFolderSize("unfiled", 0)
+
+        val testFolderName = "Krissinho's Test Folder"
```
Comment (tamaskozmer) [inline apps/student/src/androidTest/java/com/instructure/student/ui/e2e/FilesE2ETest.kt:220]: 😄

**Review by tamaskozmer — APPROVED** [review]

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Pull request merged**